src/main/java/com/example/project/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,27 @@
+package com.example.project.dto.user;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import org.hibernate.validator.constraints.Length;
```

Comment:
use jakarta.validation.constraints.Size;

Reply:
got it